site/_posts/2017-09-11-changelogs.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Last Week in ManageIQ: Why Changelogs - Cause all the Cool Kids Do It"
```

Comment:
TBH I'm all for playfulness but I think 'Cause all the Cool Kids Do It' in the title is a weird contradiction. keepachangelog.com outlines why keeping a changelog is extremely beneficial and why you should do it beyond the reasoning that 'everyone else is doing it', so the title right off the bat strikes me as just odd.

Moreover, the article is more about 'how' we keep changelogs rather than 'why' - though adding a paragraph elaborating more on why (as described on keepachangelog) might make it fit the title more.

Reply:
Actually, that's not playfulness,..it's a scientific fact. Massive government studies prove this point.
and actually, to your point about adding a paragraph about answering the "why",...
Take a look back at the title,...the "why" is answered there.

In all seriousness, keepachangelog.com does a masterful job detailing the whys, the hows, and then whens. Didn't want to plagiarize or duplicate that effort. Only looking to draw people into thinking about the subject and perhaps,...bring some more Cool Kids into the fold.

P.S. I thought the 60 urls that point back to keepachangelog.com kind of explained that you needed to go there for more details on the process. Darn,..perhaps I need another 10 urls. 😄